sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersIntegrationTestBase.java

```diff
@@ -143,4 +142,22 @@ private String redact(String content, Matcher matcher, String replacement) {
         return content;
     }
 
+    private static String getTestPhoneNumber() {
+        String defaultPhoneNumber = Configuration.getGlobalConfiguration().get("AZURE_PHONE_NUMBER", "+11234567891");
+        String testAgent = Configuration.getGlobalConfiguration().get("AZURE_TEST_AGENT");
+        
+        if (testAgent == null) {
```

Comment:
I'd rather throw an error if the test agent is not set, so that we can enforce its usage.
WDYT?

Reply:
Actually, the intention is to only use the phone number pool (and, by extension, these environment variables) when running live tests in parallel; and falling back on the existing method of passing the test number using the `AZURE_PHONE_NUMBER` variable. This way, the existing workflow is not disrupted when developing/testing locally.

What are your thoughts on this?

Reply:
If this is the case, I think it's worth verifying whether it's a live test or not in advance, if so, just return the env variable's value, otherwise check the test agent, and if it's null, throw an exception.

The rationale behind it is that I want to fail fast in case we misconfigure the live tests, and get warned about it.